web_widget_table/views/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="assets_backend" inherit_id="web.assets_backend">
+        <xpath expr=".">
+            <link href="/web_widget_table/static/css/web_widget.css"
+                  rel="stylesheet"/>
+            <link href="https://cdn.datatables.net/v/dt/dt-1.10.18/datatables.min.css"
```

Comment:
I don't know what is the official posture in OCA for the CDN usage. But the modules that i can see don't use CDN.
Odoo can run on environments without internet.

Please, don't use minified files... Odoo does it automatically ;) and is usefull for debugging.

Reply:
Thanks for the comment, I included all the external references directly to my application context so there are no external dependencies. However, these are big files, I'm wondering that if this widget was bigger and used many packages, would I need to include all the packages here? Do you know if we can use a package manager?

Reply:
@pedrobaeza @tarteo @simahawk @yajo can help you better.

Reply:
:zipper_mouth_face: Don't ask about JS package managers! This could become a emacs vs vim situation! :bomb: 

:laughing: Jokes apart, it's a pending subject where there's no consensus here at @OCA, so we've just continued Odoo's methodology of just include the external libraries, unminified, under `/static/lib/` and assets from there. Odoo itself will take care of minification.